src/nodemanager.rs

```diff
@@ -393,18 +395,28 @@ impl NodeManager {
 
         let fee_estimator = Arc::new(MutinyFeeEstimator::default());
 
-        // load lsp client, if any
-        let lsp_client: Option<LspClient> = match lsp_url.clone() {
+        // load lsp clients, if any
+        let lsp_clients: Vec<LspClient> = match lsp_url.clone() {
             // check if string is some and not an empty string
-            Some(lsp_url_ref) if !lsp_url_ref.is_empty() => {
-                // TODO what to do if the getinfo call fails?
-                // is erroring out completely okay?
-                let lsp_client = LspClient::new(lsp_url_ref)
-                    .await
-                    .map_err(|_| MutinyError::LspFailure)?;
-                Some(lsp_client)
+            Some(lsp_urls) if !lsp_urls.is_empty() => {
+                let urls: Vec<&str> = lsp_urls.split(',').collect();
```

Comment:
why not a vec of URLs?

Reply:
because `LspClient` takes a string

Reply:
Would be cleaner if `lsp_urls` was `Vec<String>` or &str instead. This `split` happens before turning into `LspClient` anyways.